server.go

```diff
@@ -232,9 +233,9 @@ type server struct {
 
 	readPool *pool.Read
 
-	// globalFeatures feature vector which affects HTLCs and thus are also
-	// advertised to other nodes.
-	globalFeatures *lnwire.FeatureVector
+	// featureMgr dispatches feature vectors for various contexts within the
+	// daemon.
+	featureMgr *feature.Manager
```

Comment:
we will need to make one for each new context, but that is my plan for invoices specifically since it will be used outside of the server/main pkg

Reply:
I think it would work here though, as the `get` method is all that is used from this manager after it is created. As we thread this into other packages, then we wouldn't need to depend on the manager struct. But no blocker, just an observation.